### PR TITLE
New package: rambox-0.5.13

### PR DESCRIPTION
**Pull request by R13N**

### CODE REVIEW

File: srcpkgs/rambox/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+revision=1
+build_style=fetch
+short_desc="Free and Open Source messaging and emailing app"
+maintainer="Rien Maertens <maertensrien@gmail.com>"
+license="GPL-3"
+homepage="http://rambox.pro/"
+distfiles="https://github.com/saenzramiro/rambox/releases/download/${version}/Rambox-${version}-x64.tar.gz"
+checksum="eb506db7e1b412dbdbf9e0ab69686cc10f60e3c7f2e338db5c477e45eb05d198"
+nopie=yes
+only_for_archs="x86_64"
```
Comment (Vaelatern) [inline srcpkgs/rambox/template]: Really?
This app is free and open source.
So build it from that open source.

File: srcpkgs/rambox/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+pkgname=rambox
+version=0.5.13
```
Comment (Vaelatern) [inline srcpkgs/rambox/template]: Start the template with a comment.

File: srcpkgs/rambox/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+revision=1
+build_style=fetch
```
Comment (Vaelatern) [inline srcpkgs/rambox/template]: No. It's FOSS. Treat it that way.

**Comment by Vaelatern** [comment]
Is electron. Do we care?

File: srcpkgs/rambox/template (hunk 2 above)
Comment (R13N) [reply srcpkgs/rambox/template]: Should this be a check added to `xlint`?

**Comment by R13N** [comment]
You are right about building this the FOSS way, but it is electron so building it is a hassle.
I'm going to try building an alternative app, which does the same, but is easier to build.

**Pull request closed without merging**